### PR TITLE
Fix: report alert messages are always shown

### LINKED ISSUE #683: Encoder used to show checkpoint value should always be shown

### PR DESCRIPTION
**Pull request by MatthijsSmets**
Previously alert messages for reports like: `Message is null` or `Message of type ... is encoded using ...` were only shown when clicking the `Show metadata` button.
image

Now they are always shown.
image

Closes #683

### CODE REVIEW

File: src/app/report/report-alert-message/report-alert-message.component.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  checkIfAnyMessagesPresent(): void {
+    if (ReportUtil.isCheckPoint(this.report)) {
+      this.anyMessagesPresent = !!(
+        this.report.streaming ||
+        this.report.stubbed ||
+        !this.report.message ||
+        this.report.encoding ||
+        (this.report.preTruncatedMessageLength && this.report.preTruncatedMessageLength > 0) ||
+        this.report.stubNotFound
+      );
```
Comment (Matthbo) [inline src/app/report/report-alert-message/report-alert-message.component.ts:31]: `!this.report.message` why is this turned negative? Wouldn't not having a message do the exact opposite of what you want?

Comment (MatthijsSmets) [reply src/app/report/report-alert-message/report-alert-message.component.ts:31]: Yeah the property names are a little confusing, this method sets a boolean to true if any alert messages should be shown. One of those alert messages is that the message of a report is null, thats why it checks if its null. I will change the name of the method.

Comment (Matthbo) [reply src/app/report/report-alert-message/report-alert-message.component.ts:31]: Maybe also rename `anyMessagesPresent` to `anyAlertMessagesPresent`

Comment (MatthijsSmets) [reply src/app/report/report-alert-message/report-alert-message.component.ts:31]: Yeah good point, fixed it.

**Review by Matthbo — APPROVED** [review]

**Pull request merged**